src/vm/interpreter.zig

```diff
@@ -287,7 +293,7 @@ pub const Vm = struct {
                 switch (@intFromEnum(opcode) & 0xF0) {
                     Instruction.add,
                     => if (!opcode.is64()) {
-                        result = extend(result);
+                        result = @as(u32, @truncate(extend(result)));
```

Comment:
What exactly is this supposed to do?

In your code, I think `extend` is superfluous. This should be equivalent:

```suggestion
                        result = @as(u32, @truncate(result));
```

Is that the behavior you're looking for?

Reply:
Yeah that would also work, I'll remove the extra `extend`.

Reply:
Done in: [e788642](https://github.com/Syndica/sig/pull/620/commits/e788642b29b8be40b774254aa796b0071baf97b1)